l10n_be_cooperator/reports/tax_shelter_shares_report.xml

```diff
@@ -143,78 +145,74 @@
 
                 <!-- TODO: Translate below to English -->
                 <p>
-                    L' Article 145/26, CIR 92 sur les revenus<span
-                        t-field="o.declaration_id.fiscal_year"
-                    />, connu aussi
+                    L'Article 145/26, CIR 92 sur les revenus
+                    <span t-field="o.declaration_id.fiscal_year" />, connu aussi
                     comme normative Tax Shelter, donnent droit à une réduction
-                    d’impôt à hauteur de &#160;<span
-                        t-field="o.declaration_id.tax_shelter_percentage"
-                    />
+                    d'impôt à hauteur de
+                    <span t-field="o.declaration_id.tax_shelter_percentage" />
                     du montant de la valeur des parts souscrites depuis
                     <span t-field="o.declaration_id.month_from" />
                     (et avant
                     <span t-field="o.declaration_id.month_to" />
-                    pour la déclaration des revenus &#160;<span
-                        t-field="o.declaration_id.fiscal_year"
-                    />).
+                    pour la déclaration des revenus
+                    <span t-field="o.declaration_id.fiscal_year" />).
                 </p>
 
                 <p>
                     Cette lettre vaut donc comme certificat que
                     <span t-field="o.company_id.name" />
                     remplit en effet tous les critères permettant un tel
                     avantage fiscal. Dans les grandes lignes il faut être une
-                    micro-entreprise n’ayant encore jamais distribué de
-                    dividende et qui n'est pas issue d’une fusion ou scission de
+                    micro-entreprise n'ayant encore jamais distribué de
+                    dividende et qui n'est pas issue d'une fusion ou scission de
```

Comment:
^^ (le sens du détail!)

Reply:
i prefer typographic apostrophes, but since we’re using them nowhere else in odoo, i replaced them all. 😁